### PR TITLE
Stereo: replace toe-in by off-axis method (rebased #3620)

### PR DESCRIPTION
**Pull request by sapier**

### CODE REVIEW

**Comment by sapier** [comment]
This is in fact #3620 just rebased due to the amount of time since it was requested

**Comment by Fixer-007** [comment]
Please give at least some name to this PR

**Comment by paramat** [comment]
Is the 1 approval your own?
Could you squash and add a descriptive commit message?

**Comment by sapier** [comment]
paramat content of this pull ain't my own as mentioned above work has been done by dfelinto whom did the initial pull request.
As by the time he was around noone did realize this pull IS worth merging I did rebase it.

I'd prefere not to merge the commits as each of them does different thing. Basicaly each is for a different 3d mode and (most) could be applied independently

**Comment by paramat** [comment]
I see, that's fine.

File: src/drawscene.cpp [hunk 1]
```diff
@@ -122,18 +195,24 @@ void init_texture(video::IVideoDriver* driver, const v2u32& screensize,
...
+		Hud& hud, bool draw_wield_tool, Client& client, gui::IGUIEnvironment* guienv,
+		video::SColor skycolor )
 {
 	static video::ITexture* images[2] = { NULL, NULL };
 	static v2u32 last_screensize = v2u32(0, 0);
+	scene::ICameraSceneNode *cameraNode = camera.getCameraNode();
 
 	video::ITexture* image = NULL;
 
+	float halfInterocularDistance = g_settings->getFloat("3d_paralax_strength");
```
Comment (SmallJoker) [inline src/drawscene.cpp]: This setting `3d_paralax_strength` is documented as `3d_parallax_strength` (two `l`s)  in the minetest.conf.example file.
It should be corrected here and in the default settings aswell.

File: src/drawscene.cpp [hunk 2]
```diff
@@ -31,30 +31,99 @@ typedef enum {
...
 
+/* analog to D3DXMatrixPerspectiveOffCenterLH, not glFrustum*/
+void getProjectionFrustum(
+        const float left,
+        const float right,
+        const float bottom,
+        const float top,
+        const float cnear,
+        const float cfar,
+        irr::core::matrix4& matrix)
```
Comment (SmallJoker) [inline src/drawscene.cpp]: Spaces.

**Comment by paramat** [comment]
Adoption needed, close in 1 month if no interest.

**Comment by paramat** [comment]
Neglected, closing.

**Pull request closed without merging**